install_config/storage_examples/gluster_example.adoc

```diff
@@ -13,528 +14,39 @@ toc::[]
-# oc create -f gluster-service.yaml
-endpoints "gluster-cluster" created
-----
-====
-
-Verify that the service was created:
-
-====
-----
-# oc get service gluster-cluster
-NAME                CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
-gluster-cluster     10.0.0.130   <none>        1/TCP     9s
-
-----
+All `oc` commands are executed on the {product-title} master host.
 ====
 
-[[complete-example-using-gusterfs-creating-the-persistent-volume]]
-== Creating the Persistent Volume
-Next, before creating the PV object, define the persistent volume in
-{product-title}:
+[[prerequisites]]
+== Prerequisites
 
-.Persistent Volume Object Definition Using GlusterFS
+include::install_config/persistent_storage/topics/glusterfs_prereqs_software.adoc[]
 
-[source,yaml]
-----
-apiVersion: v1
-kind: PersistentVolume
-metadata:
-  name: gluster-pv   <1>
-spec:
-  capacity:
-    storage: 1Gi     <2>
-  accessModes:
-  - ReadWriteMany    <3>
-  glusterfs:         <4>
-    endpoints: gluster-cluster <5>
-    path: /HadoopVol <6>
-    readOnly: false
-  persistentVolumeReclaimPolicy: Retain <7>
-----
-<1> The name of the PV, which is referenced in pod definitions or displayed in
-various `oc` volume commands.
-<2> The amount of storage allocated to this volume.
-<3> `*accessModes*` are used as labels to match a PV and a PVC. They currently
-do not define any form of access control.
-<4> This defines the volume type being used. In this case, the *glusterfs*
-plug-in is defined.
-<5> This references the endpoints named above.
-<6> This is the Gluster volume name, preceded by `/`.
-<7> The volume reclaim policy `Retain` indicates that the volume will be
-preserved after the pods accessing it terminates. For GlusterFS, the accepted
-values include `Retain`, and `Delete`.
+include::install_config/persistent_storage/topics/glusterfs_security_selinux.adoc[]
 
-Save the PV definition to a file, for example *_gluster-pv.yaml_*,
-and create the persistent volume:
+[[static-provisioning]]
+== Static Provisioning
 
-====
-----
-# oc create -f gluster-pv.yaml
-persistentvolume "gluster-pv" created
-----
-====
-
-Verify that the persistent volume was created:
-
-====
-----
-# oc get pv
-NAME         LABELS    CAPACITY   ACCESSMODES   STATUS      CLAIM     REASON    AGE
-gluster-pv   <none>    1Gi        RWX           Available                       37s
-----
-====
+include::install_config/persistent_storage/topics/glusterfs_static_provisioning.adoc[]
 
-[[complete-example-using-gusterfs-creating-the-persistent-volume-claim]]
-== Creating the Persistent Volume Claim
-A persistent volume claim (PVC) specifies the desired access mode and storage
-capacity. Currently, based on only these two attributes, a PVC is bound to a
-single PV. Once a PV is bound to a PVC, that PV is essentially tied to the PVC's
-project and cannot be bound to by another PVC. There is a one-to-one mapping of
-PVs and PVCs. However, multiple pods in the same project can use the same PVC.
+[[using-the-storage]]
```

Comment:
@jarrpa - this anchor tag is used in the gluster_dynamic_example.adoc file as well. Can you change this to be more specific to this topic?